.github/workflows/elixir.yml

```diff
@@ -183,8 +177,5 @@ jobs:
     steps:
       - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579
       - uses: ./.github/actions/gradle_cache
-      - uses: ./.github/actions/cargo_home_cache
-      - uses: ./.github/actions/cargo_target_dir_cache
```

Comment:
Only `../../gradlew test_ockam_vault_software` requires rust caches as it compiles crates.